xrpl_api/src/api/subscribe.rs

```diff
@@ -122,10 +121,11 @@ pub struct ValidationReceivedEvent {
     // #TODO add missing fields
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Deserialize)]
 pub struct TransactionEvent {
-    #[serde(rename = "type")]
-    pub event_type: String,
```

Comment:
Removed in order to support tagged deserialization when using these events in a `Deserialize`able enum.